File: curso-python/ex071.py
print("="*50)
print("BANCO CEV".center(50))
print("="*50)

cedulas = [100,50,20,10,5,2,1]
while True:
    saque = int(input("qual valor você quer sacar?: R$"))
    if saque < 1:
        print("ERRO! Digite um valor válido")
    
    for valor in cedulas:
        quantidade = saque // valor
        saque = saque - (quantidade * valor)
        if quantidade > 0:
            print(f"total de {quantidade} cédula(s) de R${valor}")
    break

File: javascript-exercicios/ex071.js
const prompt = require("prompt-sync")()

console.log("--------------------------------------")
console.log("              BANCO CEV               ")
console.log("--------------------------------------")

let cedulas = [100,50,20,10,5,2,1]

while(true){
    let saque = Number.parseInt(prompt("qual valor você quer sacar?: R$"))
    if (saque < 1){
        console.log("ERRO! Digite um valor válido")
    }

    for (let valor of cedulas){
        let quantidade = Math.floor(saque / valor)
        saque = saque - (quantidade * valor)
        if(quantidade > 0){
            console.log(`total de  ${quantidade} cédula(s) de R$${valor}`)
        }
    }
    break
}